holoviews/core/util.py

```diff
@@ -824,6 +832,16 @@ def wrap_tuple_streams(unwrapped, kdims, streams):
     return tuple(substituted)
 
 
+def drop_streams(streams, keys, kdims):
+    """
+    Drop any dimensionsed streams from the keys and kdims.
```

Comment:
I still think it is a little odd to put the keys (typically a very long list) before the kdims (much shorter). The order that feels (slightly) more natural to me for the signature is ``(streams, kdims, keys)``. As there can be a long list of streams, the next best is probably ``(kdims, streams, keys)``. 

Just a minor point that you may ignore if you wish!